loggo/loggo.py

```diff
@@ -252,9 +248,11 @@ def full_decoration(*args: Any, **kwargs: Any) -> Any:
             Args and kwargs are for/from the decorated function
             """
             bound = self._params_to_dict(function, *args, **kwargs)
-            # bound will be none if inspect signature binding failed. in this
-            # case, error log was created, raised if self.raise_logging_errors
             if bound is None:
```

Comment:
if `bound` could never be `None`, why this check?

Reply:
Probably a bug. But now it can be `None` with the changes made to `_params_to_dict`.